### ISSUE TITLE
Add Image upload functionality in Create Event 

### ISSUE THREAD

**Issue by duplixx** [PalisadoesFoundation/talawa-admin#1437]
**Is your feature request related to a problem? Please describe.**
Previously, when creating an event, the image was not included in the schema on the form. However, with the recent update to the Talawa API on the backend, image support has been added for creating events. 

https://github.com/PalisadoesFoundation/talawa-api/pull/1675

**Describe the solution you'd like**
I propose implementing image upload functionality to enhance the event creation process.

**Describe alternatives you've considered**

**Approach to be followed (optional)**
A clear and concise description of approach to be followed.

**Additional context**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by duplixx** [comment]
@palisadoes please assign me !!

**Comment by duplixx** [comment]
Hello @Cioppolo14, I'm eager to work on this issues since I've added this functionality to the backend. I really want to implement it in the frontend as well. I'll be waiting to start once the backend PR is merged in Talwa-api.

**Comment by Ajen07** [comment]
@Cioppolo14  can I get assigned to this issue ?

**Comment by abhisheksharma010** [comment]
Hello, can you assign this issue to me? I can easily resolve this.

**Comment by palisadoes** [comment]
Unassigning due to inactivity and/or no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue.

**Comment by kanhaiya04** [comment]
@palisadoes, I am interested in taking on this task since no one is currently working on it.

**Comment by Cioppolo14** [comment]
@kanhaiya04 You currently have 2 issues. Our policy is to assign no more than two issues to each contributor across all repositories. This way everyone gets a chance to participate in the projects. We sometimes give exceptions for more urgent cases and sometimes we lose track, but the policy stands. You have reached your limit, please wait until your existing issues are closed before requesting more issues. You could unassign yourself from one of the other issues too.

**Comment by devsargam** [comment]
Hey, can I please work on this?

**Comment by duplixx** [comment]
> Unassigning due to inactivity and/or no submitted PR. Others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue.

Sorry its taking longer in implementing backend images support due to some failing tests i am actively working on that

**Comment by devsargam** [comment]
@duplixx how long is the backend support gonna take? or is it already done for this feature???

**Comment by duplixx** [comment]
yeah i pushed my changes it will merge soon

**Comment by devsargam** [comment]
@duplixx how long is it gonna take for the backend portion to be implemented???

**Comment by palisadoes** [comment]
1. Unassigning due to inactivity.
2. The related PR has been merged
    1. https://github.com/PalisadoesFoundation/talawa-api/pull/1675

**Comment by varshith257** [comment]
I would like to work on this issue

**Comment by varshith257** [comment]
Working on it

**Comment by pranshugupta54** [comment]
I can work on this

**Comment by pranshugupta54** [comment]
@palisadoes, after exploring this - it requires some changes in API. Currently the images are stored as 64base and not encoded properly so requires us to make changes to API. This is part of GSoC Project - `Improved File Uploads and Security`, so we should not add more image functionalities before finalising file system. This issue should either be taken under the GSoC project itself or completed after successful implementation of file system. 

We can close this for now.
